clinica/utils/ux.py

```diff
@@ -0,0 +1,86 @@
+# coding: utf8
+
+
+def print_images_to_process(list_participant_id, list_session_id):
+    """Print which images will be processed by the pipeline."""
+    from .stream import cprint
+    images_to_process = ', '.join(list_participant_id[i][4:] + '|' + list_session_id[i][4:]
```

Comment:
When you have many images to process, I wanted to shorten a little bit the text to display by display participant/session labels instead of participant/session IDs e.g.:
```
The pipeline will be run on the following 20 image(s): CLNC01|M54, CLNC01|M36, CLNC01|M18, CLNC01|M72, CLNC01|M00, CLNC02|M54, CLNC02|M36, CLNC02|M18, CLNC02|M72, CLNC02|M00, CLNC03|M54, CLNC03|M36, CLNC03|M18, CLNC03|M72, CLNC03|M00, CLNC04|M54, CLNC04|M36, CLNC04|M18, CLNC04|M72, CLNC04|M00
```
would save 160 characters and avoid redundancy.

It is only a proposition, not the final decision. So I propose to decide in a meeting :-)

PS: I will add an arbitrary threshold (`len(list_participant_id) < 25`) in order to display only the number of images.